confd/templates/gui/application.properties.tmpl

```diff
@@ -11,6 +11,11 @@ server.ssl.key-alias = kilda
 server.ssl.key-store = classpath:keystore-kilda.jks
 server.ssl.key-store-password = openkilda
 
+#SSL Victoria configurations
+#victoria.trust.store=file:./keystore-victoria-kilda.jks
```

Comment:
It is confusing for someone not familiar with the area. Please consider to add some more information such as why would one need it from this folder. Or maybe just remove comment if the usage is unlikely.

Reply:
thanks, will write more description